.github/workflows/actions.yml

```diff
@@ -73,11 +73,7 @@ jobs:
           fi
       - name: Create GitHub Release
         id: create_gh_release
-        uses: actions/create-release@v1.1.4
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # This token is provided by Actions, you do not need to create your own token
+        uses: softprops/action-gh-release@v1
         with:
-          tag_name: ${{ github.ref }}
-          release_name: ${{ github.ref }}
-          body:
-          draft: false
+          generate_release_notes: true
+          append_body: https://github.com/com-lihaoyi/requests-scala#changelog
```

Comment:
Does that mean, we copy the whole changelog here? This is not what we want. Isn't a link better, to keep it DRY and consistent?

Reply:
Nope, this would just mean that the literal link gets added to the body of the release notes.

Reply:
Oh, ok. That's fine.